src/main/java/com/streamr/broker/cassandra/CassandraStatementBuilder.java

```diff
@@ -23,7 +23,7 @@ BoundStatement eventInsert(StreamMessage msg) {
 			msg.getStreamId(),
 			msg.getStreamPartition(),
 			msg.getTimestampAsDate(),
-			msg.getSequenceNumber(),
+			(int) msg.getSequenceNumber(),
```

Comment:
Why cast to `int`?

Reply:
`streamr-docker-dev` updated to have a longer field for sequence number in Cassandra